lib/Bio/EnsEMBL/DataCheck/Checks/CompareSchema.pm

```diff
@@ -110,7 +110,7 @@ sub normalise_table_def {
   $table =~ s/ IF NOT EXISTS//gm;
 
   # Normalise case: everything after column name is upper-cased.
-  $table =~ s/^([a-zA-Z]\w+\s)(.+)/$1\U$2/gm;
```

Comment:
If we want the column to be lowered in cases, we need to remove the \U for matched group 2